Library/Formula/sourcekitten.rb

```diff
@@ -0,0 +1,16 @@
+class SourceKitten < Formula
```

Comment:
What Mike means is that the class name corresponds to the filename of the formula. If the formula is named `sourcekitten.rb`, then Homebrew will attempt to load a class named `Sourcekitten`. A CamelCased class comes from a formula with a dash or an underscore.